ssas/service/api_common_test.go

```diff
@@ -2,14 +2,15 @@ package service
 
 import (
 	"encoding/json"
-	"github.com/CMSgov/bcda-ssas-app/ssas"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/suite"
-	"gorm.io/gorm"
 	"io/ioutil"
 	"net/http"
 	"net/http/httptest"
 	"testing"
+
+	"github.com/CMSgov/bcda-ssas-app/ssas"
```

Comment:
like the import ordering! 🎉

Reply:
my IDE does this automatically 😂 can't help it 🤷